purchase_split_wizard/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Split Wizard",
+    "summary": "Wizard to split one purchase order in two purchase orders",
```

Comment:
This is not accurate. You can split a purchase order into as many purchase orders as you want, right?

Reply:
Changed